app/Http/Api/Controller/V1/UserController.php

```
@@ -34,11 +30,8 @@ public function __construct(
     )]
     public function login(UserRequest $request): Result
     {
-        return $this->success(
-            $this->passportService->login(
-                $request->input('username'),
-                $request->input('password')
-            )
-        );
+        $validated = $request->validated();
```

PHPMD (2.15.0)

Avoid unused local variables such as '$validated'. (Unused Code Rules)

(UnusedLocalVariable)